identity-iota/src/tangle/did_message_versioning.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::Error;
+
+#[derive(Copy, Clone)]
+pub(crate) enum DIDMessageVersion {
+  V1 = 1,
+}
+
+pub(crate) const CURRENT_MESSAGE_VERSION: DIDMessageVersion = DIDMessageVersion::V1;
+
+/// Checks if flag matches message_version.
+pub(crate) fn check_version_flag(flag: &u8, message_version: DIDMessageVersion) -> Result<(), Error> {
+  if message_version as u8 == *flag {
```

Comment:
Same as `DIDMessageEncoding`, this would benefit from a  `TryFrom<u8>` or `FromPrimitive` implementation.